app/scripts/models/auth_brokers/first-run.js

```diff
@@ -18,6 +18,7 @@ define([
     _iframeCommands: {
        LOADED: 'loaded',
        LOGIN: 'login',
+       VERIFY_SIGNUP: 'verify_signup',
```

Comment:
Is `verify_signup` a reasonably descriptive message, or would `must_verify_signup` be better?